user_guide/data_management_workflows.rst

```diff
@@ -0,0 +1,105 @@
+.. _user-guide-workflow-management-basics:
+
+Following a Workflow
+====================
+
+In the process of the work with the customers information saved in the CRM system can be changed: carts turn 
+into orders, what seemed to be a potential opportunity appears not to be worth effort or tuns into a successful deal, 
+customer support requests are processed and closed, and so on.
+These and other processes are reflected in OroCRM with creation of new records, change of record properties and 
+creation of new relations. (For example - a new lead appeared, and while processed, it was related to a customer and a 
+contact, and turned into a new opportunity. 
+
+In many cases success of the business depends a lot on the unity of the procedures followed all through the company.
+How many times can you call a customer? Can you close an opportunity as lost without an explanation? What should you do 
+if a customer has added some goods to the cart but hasn't ordered them?
```

Comment:
Edit: In many cases, the success of a business depends on how closely employees throughout the company follow procedures. These can include things like how many times a customer can be called, whether an opportunity can be declared lost without an explanation, and what should be done when a customer adds items to their cart, but hasn't ordered them.